docs/components/connectors/out-of-the-box-connectors/email.md

```diff
@@ -0,0 +1,695 @@
+
+This inbound connector will create a new process each time a new email will be received.
+
+- `Folder`: (Optional) Specifies the folder that the inbound connector should monitor. The default folder is INBOX.
+- `Sync Strategy`: Determines how emails are synchronized when the connector starts:
+    - `Unseen emails will be sync`: A process instance is created for every unseen email present in the folder at the
+      time the connector starts.
+    - `No initial sync. Only new emails`: The connector skips past emails and starts listening for new emails arriving
+      in the folder.
+    - `All emails will be sync`: A process instance is created for every email in the folder, regardless of their read
+      status, when the connector starts.
+- `Handling Strategy`: Defines how emails are handled after processing:
+    - `Mark as read after processing`: Emails that have been processed will be marked as read.
+    - `Do nothing`: No action will be taken on processed emails.
+    - `Delete after processing`: Emails that have been processed will be deleted from the folder.
+    - `Move to another folder after processing`: Processed emails will be moved to another specified folder.
+        - `Folder`: Indicates the destination folder where emails will be moved after processing. A new folder or folder
+          hierarchy
+          can be specified using a dot-separated path (e.g., 'Archive' or 'Projects.2023.January'). Non-existent folders
+          in the
+          path will be created automatically.
+
+### Example Response
+
+The JSON response below is an example of the data structure produced when an email triggers the creation of a process
+instance:
+
+```json
+{
+  "messageId":"messageId",
+  "fromAddresses":["example@camunda.com"],
+  "subject":"Urgent Test",
+  "size":65646,
+  "plainTextBody":"Hey how are you?\r\n",
+  "htmlBody":"<html>Hello</html>"
+}
+```
+
+This response includes essential email details such as the messageId, sender addresses, subject, size, and the content
+of the email both in plain text and HTML format. This information can be used by the process for various workflows, such
+as prioritizing tasks, content analysis, and automated responses.
+
+### Activation condition
+
+The **Activation condition** is an optional field where you can specify a Friendly Enough Expression Language (FEEL)
+expression to control when the Email Inbound Connector should trigger a process instance. This condition acts as a
+filter, allowing the process to be initiated only when certain criteria are met by the incoming email.
+
+#### Example Usage
+
+For instance, the FEEL expression =(response.subject = "urgent") will ensure that the process is only triggered if the
+subject of the incoming email matches "urgent". If this field is left blank, the process will be triggered for every
+email received by the connector.
+
+:::warning By default, the Email Inbound Connector is designed not to execute its handling strategy if it encounters an
```

Comment:
the rendering is weird there
<img width="962" alt="Screenshot 2024-09-17 at 10 10 57" src="https://github.com/user-attachments/assets/e523b34d-93ea-4cbc-b9ce-36d56b52a157">